crypto/rand/rand_lcl.h

```diff
@@ -17,21 +17,24 @@
 # include <openssl/ec.h>
 # include "internal/rand.h"
 
-/* we require 256 bits of randomness */
-# define RANDOMNESS_NEEDED (256 / 8)
+/* Amount of randomness (in bytes) we want for initial seeding. */
+# define RANDOMNESS_NEEDED              (128 / 8)
```

Comment:
That is not at all what I read in that rfc. It talks about "the amount of randomness in the very strong key to a minimum of 192 bits (96*2) is required for the year 2004".

Anyway, I've never heard that it's not required to have that much entropy to generate a good key, so please try to find a reference that actually claims that.